geode-core/src/main/java/org/apache/geode/distributed/internal/LocatorLoadSnapshot.java

```diff
@@ -497,13 +558,21 @@ private void updateMap(Map map, ServerLocation location, float load, float loadP
    * If it is most loaded then return its LoadHolder; otherwise return null;
    */
   private LoadHolder isCurrentServerMostLoaded(ServerLocation currentServer,
-      Map<ServerLocation, LoadHolder> groupServers) {
-    final LoadHolder currentLH = groupServers.get(currentServer);
+      Map<ServerLocationAndMemberId, LoadHolder> groupServers) {
```

Comment:
Not a new method but significantly changed, it should be, at leat, unit tested.
You can make it package private, annotate it with `@VisibleForTesting` and access it directly from `LocatorLoadSnapshotJUnitTest` and/or `LocatorLoadSnapshotIntegrationTest`.

Reply:
done!